projects/ccf-rui/src/app/modules/content/content.component.html

```diff
@@ -4,4 +4,11 @@
   </ccf-stage-nav>
   <mat-icon class="icon reset" (click)="resetStage()">refresh</mat-icon>
 </div>
-<ccf-store-debug></ccf-store-debug>
+<ccf-toggleable-tooltip
+    [visible]="tutorialMode"
+    [direction]="'down'"
+    [offset]="21"
+    [tooltip]="'Change view.'"
+></ccf-toggleable-tooltip>
```

Comment:
For all these with constant strings for @Inputs, use the cleaner syntax field="my string". ie:
```html
<ccf-toggleable-tooltip
    [visible]="tutorialMode"
    direction="down"
    [offset]="21"
    tooltip="Change view."
></ccf-toggleable-tooltip>
```